tron/deployerImplementation.js

```diff
@@ -0,0 +1,164 @@
+module.exports.deploy = async function(inputFile, feeLimit, energyLimit, rpcUrl, dontSendTx, tokenName) {
+
+  const TronWeb = require('tronweb');
+  const fs = require("fs");
+  const path = require('path');
+  const solc = require('solc')
```

Comment:
for tron, we need to be using `tron-solc` not the solidity compiler with Ethereum

https://github.com/tronprotocol/solidity

should be built using the specification here: 

https://github.com/TRON-US/tron-solc-bin